wordpress-mobile/WordPress-Android
# Add missing tracks events for reader post details after refactoring

Task: #12900

This PR 
- Adds tracks events which got removed from the reader post detail fragment during refactoring
- Fixes [un-bookmark event](https://github.com/wordpress-mobile/WordPress-Android/blob/fcf331afbc3da3ff6c1cb5806b87e213dfdf30ec/WordPress/src/main/java/org/wordpress/android/ui/reader/usecases/ReaderPostBookmarkUseCase.kt#L64) from other post list [here](https://github.com/wordpress-mobile/WordPress-Android/blob/3e66d51ede62282dcd2e9859cadf237366fb5b05/WordPress/src/main/java/org/wordpress/android/ui/reader/usecases/ReaderPostBookmarkUseCase.kt#L71)


**To test**

- Open a reader post
- Toggle like in the footer 
- Notice that below events are shown in the logs:
    - `🔵  Tracked: reader_article_detail_liked`
    - `🔵 Tracked: reader_article_detail_unliked`
- Toggle bookmark in the footer 
- Notice that below events are shown in the logs:
   - `🔵 Tracked: reader_post_saved, Properties: {"source":"post_details"}`
   - `🔵 Tracked: reader_post_unsaved, Properties: {"source":"post_details"}`

**Questions**

1. Should we add a new tracks event for more menu tap on reader post details?
We track below event for reader post card more menu:
`🔵 Tracked: post_card_more_tapped`
and looks like we cannot use it on the details screen.

2. Globe/ visit item from more menu tracked as `🔵 Tracked: reader_article_visited`
Globe standalone icon - not tracked (consistent with iOS) - should we add tracking same as globe/ visit item from more menu?

3. Share item from more menu tracked as  `🔵 Tracked: item_shared_reader`
Share standalone icon tracked as `🔵 Tracked: item_shared` (not tracked in iOS) - should we change it `item_shared_reader` same as share item from more menu ?



PR submission checklist:
